bamboo/templates/template.yml

```diff
@@ -29,6 +46,7 @@ Build:
         #!/usr/env/bin bash
         set -e
         docker-compose -f test/docker-compose.yml up --scale selenium-chrome=3 --scale selenium-firefox=3 --exit-code-from tests
+        docker-compose -f docker-compose-performance.yml up --force-recreate --exit-code-from sitespeed-firefox
```

Comment:
Dan zullen we enkel de exit code van firefox in rekening brengen momenteel?
We moeten dit misschien verdelen over 2 regels (dus selectief de services uit de docker compose file starten per regel) of over 2 docker-compose files?

Reply:
Goed punt, ik pas het aan.